apps/server/prisma/seed.ts

```diff
@@ -1,47 +1,53 @@
-import { PrismaClient } from '@prisma/client'
-const prisma = new PrismaClient()
+import { PrismaClient } from '@prisma/client';
 import fsPromises from 'fs/promises';
 import path from 'path';
 import ProgressBar from 'progress';
 
+const prisma = new PrismaClient();
+
 async function main() {
     const filePath = path.join(process.cwd(), 'prisma/ingredients.json');
     const jsonData = await fsPromises.readFile(filePath);
     const IngredientsData = JSON.parse(jsonData.toString());
     const ingredients = IngredientsData.ingredients;
     const totalIngredients = ingredients.length;
 
-    await prisma.ingredients.deleteMany()
+    await prisma.ingredients.deleteMany(); // Delete existing records
+
     console.log('Deleted records in ingredients table');
-    var bar = new ProgressBar('  inserting [:bar] :rate/bps :percent :etas', {
+
+    const bar = new ProgressBar('  inserting [:bar] :rate/bps :percent :etas', {
         complete: '=',
         incomplete: ' ',
         width: 20,
-        total: totalIngredients
-      });
-    
-    const createIngredient = async (ingredient: { name: any; imageName: any; }) => {
-        await prisma.ingredients.create({
-            data: {
-                name: ingredient.name,
-                imageName: ingredient.imageName,
-            },
-        });
-        bar.tick();
+        total: totalIngredients,
+    });
+
+    const createIngredient = async (ingredient : any) => {
+        try {
+            await prisma.ingredients.create({
+                data: {
+                    name: ingredient.name,
+                    imageName: ingredient.imageName,
+                },
+            });
+            bar.tick();
+        } catch (error) {
+            console.error(`Error inserting ingredient: ${error.message}`);
+        }
     };
 
     const insertIngredients = async () => {
-        const promises = ingredients.map((ingredient: { name: any; imageName: any; }) => createIngredient(ingredient));
+        const promises = ingredients.map((ingredient :any) => createIngredient(ingredient));
         await Promise.all(promises);
         console.log("All ingredients inserted successfully.");
     };
 
-    insertIngredients();
+    await insertIngredients(); // Await the insertion function
+
+    await prisma.$disconnect();
 }
 
-main().catch(e => {
+main().catch((e) => {
```

Comment:
We don't need this `process.exit(1);`?, they say it might create problem
please investigate, if this is needed

Reply:
we were cloing the conect when everything was done?
```
.finally(async () => {
    await prisma.$disconnect();
})
```

Reply:
I actually didn't wanted to close the connection, to find the bug. But the bug is not actually inside the seed.ts but in .env